uiplib/uipImage.py

```diff
@@ -0,0 +1,42 @@
+"""A class that handles editing of images before applying as wallpaper."""
+
+import os
+
+from PIL import Image, ImageFilter
+
+
+class UipImage:
+    """Class that holds the image as well as the functions used to edit it."""
+
+    def __init__(self, image_path):
+        """Initialize class."""
+        self.image_path = image_path
+        self.original_path = image_path
+        self.image = Image.open(image_path)
+        self.edited = True
+        self._blur = False
+        self.blur_radius = 0
+
+    def blur(self, radius):
+        """Save blur pamaters as well as returns a blurred image."""
+        self.edited = True
```

Comment:
this isnt redundant as we're going to have many editing options other than blur right? like add filter and so on?